index.html

```diff
@@ -39,7 +39,196 @@ <h1 class="my-name">Bassem Shams</h1>
         </p>
         <button class="button">Start collaboration</button>
       </section>
-      <footer class="lower"></footer>
+      <section class="lower"></section>
+      <!--the second part of the project-->
+      <nav class="container-projects">
+        <h2 class="header-project">projects</h2>
+        <section class="project">
+          <nav>
+            <img
+              class="image-project"
+              src="./assests/images/Rectangle21.png"
+              alt="project image"
+            />
+          </nav>
+          <nav class="details-project">
+            <p class="name-project">project name goes here</p>
+            <ul class="lang-group">
+              <li class="lang-project"><span>HTML/CSS</span></li>
```

Comment:
- [ ] Kindly wrap the works section in a `<section>` tag instead of a `<nav>` tag.
The `<nav>` tag defines a set of navigation links. Notice that NOT all links of a document should be inside a `<nav>` element. The `<nav>` element is intended only for major block of navigation links like the toolbar.

You can read more on semantic elements [here](https://www.w3schools.com/html/html5_semantic_elements.asp)